packages/gui/src/electron/main.tsx

```diff
@@ -225,8 +246,8 @@ if (!handleSquirrelEvent()) {
       function getRemoteFileSize(urlLocal: string): Promise<number> {
         return new Promise((resolve, reject) => {
           axios({
-            method: 'get',
-            url: urlLocal,
+            method: 'HEAD',
+            url,
```

Comment:
This breaks fetching metadata because it's referencing the wrong variable.

```suggestion
            url: urlLocal,
```